backend/tests/unit/api/Controllers/Tools/ImportControllerTest.cs

```diff
@@ -759,6 +761,85 @@ public void ImportProjects_WithNoNote_Success()
             project.Snapshots.Should().BeEmpty();
             project.Tasks.Should().BeEmpty();
         }
+
+
+        public static IEnumerable<object[]> ProjectActivity =>
+            new List<object[]>
+            {
+                new object[] { "Completed Deal", "SPL" },
+                new object[] { "Contract in Place", "SPL" },
+                new object[] { "On Market", "SPL" },
+                new object[] { "Pre-Market", "SPL" },
+                new object[] { "GET LAST ONE", "SPL" },
+                new object[] { "", "SPL" },
+                new object[] { null, "SPL" }
+            };
+
+        /// <summary>
+        /// Based on the specified 'Activity' determine the appropriate Workflow.
+        /// </summary>
+        /// <param name="activity"></param>
+        /// <param name="expectedWorkflow"></param>
+        [Theory]
+        [MemberData(nameof(ProjectActivity))]
+        public void ImportProjects_ActivityToWorkflow(string activity, string expectedWorkflow)
+        {
+            // Arrange
+            var helper = new TestHelper();
+            var user = helper.CreateForPermission(Permissions.SystemAdmin, Permissions.PropertyAdd, Permissions.AdminProperties);
+            var controller = helper.CreateController<ImportController>(user);
+
+            var agency = EntityHelper.CreateAgency(1, "TEST");
+            var tier = EntityHelper.CreateTierLevel(1, "TIER");
+            var workflows = new[] { EntityHelper.CreateWorkflow(1, "ERP"), EntityHelper.CreateWorkflow(2, "SPL") };
+            workflows[0].SortOrder = 1;
+            workflows[1].SortOrder = 2;
+            var status = EntityHelper.CreateProjectStatus("STATUS", "TEST");
+            workflows[0].AddStatus(status, 1);
+            workflows[1].AddStatus(status, 2);
+            var risk = EntityHelper.CreateProjectRisk("Green", "GREEN", 0);
+
+            var model1 = new Model.ImportProjectModel()
+            {
+                ProjectNumber = "TEST-00001",
+                Description = "Model 1",
+                Agency = "TEST",
+                Activity = activity,
+                Status = "TEST",
+                Risk = "GREEN",
+                ReportedFiscalYear = 2020,
+            };
+
+            var adminServiceMock = helper.GetService<Mock<IPimsAdminService>>();
```

Comment:
not convinced the mock needs to be part of the test (could be in a before block), but since there is only the one test probably not a big deal.

Reply:
The mock is part of the `Arrange`.  It's necessary to setup everything for the test.  Sometimes it makes sense to setup things for a number of tests, but that can also be problematic as there is no guarantee on thread-safe and order the tests are run.  Generally best practice is to keep each test independent.